syntax = "proto3";
option cc_enable_arenas = true;

import "ydb/library/yql/providers/dq/api/protos/service.proto";
import "ydb/library/yql/dq/proto/dq_tasks.proto";

package NFq.NProto;

message TGraphParams {
    string GraphId = 1;
    repeated NYql.NDqProto.TDqTask Tasks = 2;
    map<int64, bytes> StageProgram = 12;
    uint64 SourceId = 3;
    bytes ResultType = 4;
    repeated string Columns = 5;
    string Session = 6;
    repeated Yql.DqsProto.TAttr Settings = 7;
    map<string, string> SecureParams = 8;
    string YqlText = 9;
    string QueryAst = 10;
    string QueryPlan = 11;
}
